samples/XAMLCatalog/XAMLCatalog.vsimporter/XAMLCatalog-WinStore10/XAMLCatalog-Debug-xcvars.txt

```diff
@@ -85,15 +85,15 @@ PRODUCT_BUNDLE_IDENTIFIER = Microsoft.XAMLCatalog
 PRODUCT_NAME = XAMLCatalog
 PRODUCT_TYPE = com.apple.product-type.application
 PROJECT = XAMLCatalog
-PROJECT_DIR = I:/enl/vso_1_WinObjC/samples/XAMLCatalog
-PROJECT_FILE_PATH = I:\enl\vso_1_WinObjC\samples\XAMLCatalog\XAMLCatalog.xcodeproj
+PROJECT_DIR = E:/GIT_REPOS/winobjc/samples/XAMLCatalog
+PROJECT_FILE_PATH = E:\GIT_REPOS\winobjc\samples\XAMLCatalog\XAMLCatalog.xcodeproj
 PROJECT_NAME = XAMLCatalog
 PROJECT_TEMP_DIR = sbuild/XAMLCatalog.sbuild
 PROJECT_TEMP_ROOT = sbuild
 PUBLIC_HEADERS_FOLDER_PATH = XAMLCatalog.app/Public
-SDKROOT = I:/enl/vso_1_WinObjC
-SOURCE_ROOT = I:/enl/vso_1_WinObjC/samples/XAMLCatalog
-SRCROOT = I:/enl/vso_1_WinObjC/samples/XAMLCatalog
+SDKROOT = E:/GIT_REPOS/winobjc
+SOURCE_ROOT = E:/GIT_REPOS/winobjc/samples/XAMLCatalog
```

Comment:
>E:/GIT_REPOS/winobjc/ [](start = 14, length = 21)

also this file. seemed related to different user config. should we check this file in at all?